### DISCUSSION TITLE
iModel vs IFC

### DISCUSSION THREAD

**Discussion by j209** [iTwin/itwinjs-core#891] [Q&A] [answered]
What is different between imodel and [IFC](https://en.wikipedia.org/wiki/Industry_Foundation_Classes) and how we can compare them?

**Comment by kabentley** [comment, marked as answer]
IFC is a set of schemas and a file format, intended for CAD/BIM data exchange between programs that (typically) store their data in some other format, via export/import.

iModel is a distributed database that is the permanent repository for Infrastructure Digital Twins. An iModel is queryable, editable, and open (that is, it can be used without requiring a license.) [iTwin.js](http://www.itwinjs.org) is an **open source** library for creating, synchronizing, visualizing, and querying IDTs that combine engineering data (in iModels), with reality data, GIS data, IT databases, and IOT devices into a single user experience. 

A strong and unique virtue of iModels is their distributed nature. That is, a single iModel may exist with copies in many places at the same time, each as-of some revision state. They are synchronized via changesets, managed through iModelHub. In this manner it is possible to have infinitely scalable web/cloud processing, as well as disconnected desktop and mobile programs, all written in the same codebase.
